### ISSUE TITLE
Pictures

### ISSUE THREAD

**Issue by chrispappas007** [chrispappas007/kyarch#3]
I had originally planned to add a picture or two to each synopsis. However, as I was showing a friend my map (who's a graphic artist), he suggested maybe not adding any images to keep it clean. Again, your thoughts? I may just add some pictures in and see how I feel.

**Comment by chrispappas007** [comment]
Added an image for each synopsis but need to tweak the div so it actually displays. will get to that tomorrow.

**Comment by chrispappas007** [comment]
typo with the image tag is fixed now. They load below the text, which is not what I want. This vexes me to no end...

**Comment by rgdonohue** [comment]
Yeah, layout and CSS classes can be super annoying. 
You had an opening `<b>` tag that wasn't closed in the header. Also, it seems to me that Assembly's columns don't account for margin on the x axis. So col-9 + col-3 + mx18 pushes the col-3 down to the next line. Booo ...
 
Try this?

```html
    <div class='col col--12 col--9-ml viewport-twothirds viewport-full-ml'>
      <div id="map" class='viewport-full viewport-twothirds-ml'></div>
      <div class='px12 py12 color-white bg-gray viewport-full viewport-third-ml scroll-auto'>
        <div id="summary-info" class='col col--9-ml fl'>
          <div id="time-period">
            <h1>People have lived in Kentucky for over 12,000 years. Use the slider on the left to learn about the past!</h1>
          </div>
          <div id="period-summary" class='txt-ul txt-normal'>
            <h2></h2>
          </div>
          <div id="period-link">
            <h2></h2>
          </div>
        </div>
        <div id="image" class='col col--3-ml my12 fr'>
        </div>
      </div>
    </div>
```

**Comment by rgdonohue** [comment]
That may not be ideal, but i'm using that `fl` to float the summary-info to the left and `fr` to float the image to the right.

**Comment by chrispappas007** [comment]
Yeah, the image just kinda hangs down and off to the right, but still below the text. Ideally, I would like it to be on the right side of the text. I may keep poking at this, but for now I'm going to let the pictures hang out below the text. Assembly was an easy grab to get started, but in hindsight it's more limiting than I anticipated.

**Comment by rgdonohue** [comment]
Obviously, we need to sort out the layout issue at the bottom. While getting the picture to float to the right of the text may help, it's still a little disjointed from the temporal categories the information/image is associated with. Would be nice to visually connect these better. Such as displaying a similar color as the temporal within the related info.

You could re-consider the layout and not placing the text/image at the bottom of the map (which currently requires some scrolling for smaller screens). An alternative would be to place a little question mark icon next to each of the period names next to the slider and have the text and image open in a [modal of some kind](https://www.w3schools.com/howto/howto_css_modals.asp).

**Comment by chrispappas007** [comment]
I agree with the idea of using a modal of some sort. What I've been trying to decide is where to have the call to click live. I'm not loving the idea of the button being in the sidebar because I want the user the see the site distribution for each time period before they click for the info. I asked a few different folks around the office their thoughts and got a few suggestions that I'm going to play with tomorrow. Let's see how bad I can break this thing!

**Comment by rgdonohue** [comment]
I was thinking just a little trigger off to the right of the name of the period, so it wouldn't interfere with the general UI. Something like: 
![image](https://user-images.githubusercontent.com/1952205/30764731-8cda3036-9fa9-11e7-82d5-29541575c697.png)

**Comment by chrispappas007** [comment]
Modal buttons added, modal boxes created. BUT, 2 issues: 1- I can only get the historic modal box to close. I think there is something funky with how the span tag for the close button is being called, and yet I also can't get the non-historic modals to close when I click off them (which is not dependent on the span tags). 2- How to get the boxes to display on top of the map? They are ordered to sit on top, and the inspector shows them on top, but they display behind the map. When I changed the map's z-index, the boxes sit on top by the map loses is functionality. Suggestions??

**Comment by chrispappas007** [comment]
I'm thinking I'm going to need to add a package like this: https://www.npmjs.com/package/leaflet-modal

**Comment by chrispappas007** [comment]
or bootstrap??? http://gregorthemapguy.blogspot.com/2016/02/some-leaflet-controls-for-modal-dialogs.html

**Comment by rgdonohue** [comment]
Since you're already using Assembly.css, you wouldn't want to also integrate Bootstrap (which is largely another CSS framework). All the competing class rules would drive you crazy.

The solution in [this pull request seems to work well](https://github.com/chrispappas007/kyarch/pull/11).

**Comment by chrispappas007** [comment]
Yeah, once I sat down and started going through the bootstrap API I released that it wouldn't end well. I really like the way this looks now. I see what you did with the col formatting to get the pictures to load on the side in the paleo box - sweet! I'll go in and get the rest sorted. And I see what you mean about the single modal call. I had initially been unable to get it select the right modal with my initial function so I ended up with the individual calls - your way is much better! I totally see what you mean about using the class call instead. Thanks!
